### PR TITLE
Add Github action for testing

### PR DESCRIPTION
**Pull request by echo-bravo-yahoo**
Ticket(s): FE-6016

## Problem
We currently do build/test in CircleCI, out of reach of any open source contributors.

## Solution
Test in a github action.

## Result
Contributors have a better experience w/viewing test results.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Test
+
+on:
+  push:
+    branches: [ "main", "v3" ]
+  pull_request:
+    branches: [ "main", "v3" ]
```
Comment (echo-bravo-yahoo) [inline .github/workflows/node.js.yml:10]: the "v3" branch inclusion is temporary while we're working on the next version of the CLI

Comment (ecooper) [reply .github/workflows/node.js.yml:10]: Do we want to run this and circle on main while we're working out of v3?

**Review by ecooper — APPROVED** [review]
LGTM

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (echo-bravo-yahoo) [reply .github/workflows/node.js.yml:10]: good point - let me turn it off for main

**Pull request merged**